docs/markdown/snippets/rustc-ndebug.md

```diff
@@ -0,0 +1,5 @@
+## Rust now supports the b_ndebug option
+
+Which controls the `debug_assert!()` macro. This macro is roughly equivalent to
```

Comment:
Technically it controls the `debug_assertions` config option. That in turn does control the `debug_assert!()` macro, but developers are also free to put `#[cfg(debug_assertions)]` elsewhere to have debugging code in development but not in production builds.

https://doc.rust-lang.org/reference/conditional-compilation.html#debug_assertions

Reply:
Is the updated snippet better? I actually didn't know that it controled a `cfg()`, that actually makes it really useful since we can then ensure that C and Rust have the same behavior

Reply:
LGTM